sdk/typescript/src/signers/txn-data-serializers/txn-data-serializer.ts

```diff
@@ -33,6 +33,24 @@ export interface PayTransaction {
   gasBudget: number;
 }
 
+export interface PaySuiTransaction {
```

Comment:
Do you mind copying the comments in https://github.com/gegaowp/sui/blob/d22d6d721880850fa4c0a130702c3d4ae992b6e7/crates/sui-json-rpc/src/api.rs#L241 over here? Otherwise it's hard for the users of the TS sdk to understand how these endpoints work

Reply:
will do, thanks!